### DISCUSSION TITLE
Allow text to stay on screen as you work in other applications?

### DISCUSSION THREAD

**Discussion by ColbyStewart** [pubpub-zz/ppInk#213] [Ideas]
Hey!

I work for a manufacturer and manage projects for lighting (dealing with blueprints, calculators, CAD, etc.). As I work, much of the time there are multiple windows and data I'm referencing at once, and usually the one window I wish I could avoid is my email/outlook window. However, there is often some data in the email request that I need to keep referencing, but I wish I could just pin the relevant text on top of my screen with a semi-transparent (or semi-opaque) text, as I work with other applications. So essentially, if it just had a shortcut to type/paste text on screen (and I could move it to the corner of the screen where it fits best), and have it remain on screen until I return and delete it, that would be a perfect feature for me. There could be hidden options for color, font, etc. in the options menu. Just a simple feature, but it would be a huge benefit to my team at work.

Would this be possible? Or is this available in any other application that I'm not aware of?

Thanks!
- Colby

**Comment by pubpub-zz** [comment]
Hi @ColbyStewart,
 I have some trouble to see exactly what you are looking for. Are you using ppInk for screen annotation or real-time demonstration? From my understanding, I can remind you with a few features:
a) ppInk can be started in a window mode : in this case ppInk will allow drawing and capture on only the window mode : for this you just engage a long click on pointer mode icon : you can that way leave uncovered Outlook and just go there and copy text easily.
b) You can draw some filled rectangle to cover your window, and just use the visible On/Off toggle (eye icon) to get access you information
c) you can prepare some layout, save them and then load then to get a template.

else What I can imagine is to use the rest api, that you could call with a program shortcut (using curl.exe as main program and adequate command line) which you could activate from the taskbar (if stuck), or using a hotkey.
